src/main/java/thePackmaster/SpireAnniversary5Mod.java

```diff
@@ -378,6 +378,9 @@ public void receiveAddAudio() {
     @Override
     public void receiveOnBattleStart(AbstractRoom room) {
         pandaList.clear();
+        if (p().masterDeck.contains(new Auctioneering()) || p().masterDeck.contains(new Harvest()) || p().masterDeck.contains(new NetFishing()) || p().masterDeck.contains(new IronMining()) || p().masterDeck.contains(new LumberFelling())) {
```

Comment:
I don't remember how contains checks for equality. Is a newly created instance equal?
If it is, you don't have to change this, just a small tip. You could've created an interface `VictoryCard` (and implemented it on these cards) and just iterated through your deck and checked if any card was instanceOf that interface

Reply:
ok gonna see about fixing that